project_scrum/views/project_project.xml

```diff
@@ -5,7 +5,6 @@
         <field name="name">project.project.tree</field>
         <field name="model">project.project</field>
         <field name="inherit_id" ref="project.view_project" />
-        <field name="mode">primary</field>
```

Comment:
this applies the rename of the project manager to all projects

Reply:
Yes, that was the intention of the initial author as I understand (given the fact that the `use_scrum` is there as well), that's why there is "Scrum Master/Project Manager" depending on which one applies.